milestones.go

```diff
@@ -212,3 +212,34 @@ func (s *MilestonesService) GetMilestoneIssues(pid interface{}, milestone int, o
 
 	return i, resp, err
 }
+
+// GetMilestoneMergeRequestsOptions represents the available GetMilestoneMergeRequests() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/milestones.html#get-all-merge-requests-assigned-to-a-single-milestone
+type GetMilestoneMergeRequestsOptions ListOptions
+
+// GetMilestoneMergeRequests gets all merge requests assigned to a single project milestone.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/milestones.html#get-all-merge-requests-assigned-to-a-single-milestone
+func (s *MilestonesService) GetMilestoneMergeRequests(pid interface{}, milestone int, opt *GetMilestoneMergeRequestsOptions, options ...OptionFunc) ([]*Issue, *Response, error) {
```

Comment:
This call doesn't return issues right? So guess it needs to return a different type...

Reply:
Woops!

Reply:
Fixed